### PR TITLE
2348 Add STRIVE resistance overlay to new vector overlay group

### PR DESCRIPTION
**Pull request by PShoe**
### Changes:
- added new STRIVE_Vector_Data overlay group
- added map overlay PNG_STRIVE_Resistence 'Resistance' to group

---

### CODE REVIEW

File: packages/database/src/migrations/20210602005250-AddStriveResistanceOverlay-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const mapOverlay = {
+  id: 'PNG_STRIVE_Resistence',
```
Comment (billli0) [inline packages/database/src/migrations/20210602005250-AddStriveResistanceOverlay-modifies-data.js:20]:
```suggestion
  id: 'PNG_STRIVE_Resistance',
```

**Review by billli0 — CHANGES_REQUESTED** [review]
Looks good to me! Just one typo

File: packages/database/src/migrations/20210602005250-AddStriveResistanceOverlay-modifies-data.js (hunk 1 above)
Comment (PShoe) [reply packages/database/src/migrations/20210602005250-AddStriveResistanceOverlay-modifies-data.js:20]: good catch! spelling is not my strength..

**Review by billli0 — APPROVED** [review]
LGTM@@!

**Pull request merged**